src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ProjectPackagesPrintUtility.cs

```diff
@@ -21,10 +22,17 @@ internal static class ProjectPackagesPrintUtility
         /// <param name="transitive">Whether include-transitive flag exists or not</param>
         /// <param name="outdated">Whether outdated flag exists or not</param>
         /// <param name="autoReferenceFound">An out to return whether autoreference was found</param>
-        internal static void PrintPackages(IEnumerable<FrameworkPackages> packages,
-           string projectName, bool transitive, bool outdated, out bool autoReferenceFound)
+        /// <param name="deprecatedFound">An out to return whether deprecated package version was found</param>
+        internal static void PrintPackages(
+            IEnumerable<FrameworkPackages> packages,
+            string projectName,
+            bool transitive,
+            bool outdated,
+            out bool autoReferenceFound,
+            out bool deprecatedFound)
```

Comment:
I didn't want to break the current pattern. But if we are to return something, I'd prefer a real type instead of having to deal with `tuple.Item1` or `tuple.Item2` in the calling code.

Reply:
Updated in https://github.com/NuGet/NuGet.Client/pull/2928/commits/1b63faed2f4013907bf61a3a9f8c90a02aa0aeab